src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -81,16 +87,21 @@ public static Sentiment getSentiment(String projectID, String lyrics) throws Ill
   }
 
   /**
-   * From every entity generated by the API we get only part of the information (name and salience)
-   * and store the new SongEntities into a new simplified list. 
+   * From every entity generated by the API we get only part of the information 
+   * (name, salience, type and wikiLink) and store the new SongEntities into a new simplified list. 
+   * 
+   * We round the salience value to allow us to actually test these methods.
+   * In the beginning, every entity has only one type, so we just populate the set
+   * with only one string.
    */
   public static List<SongEntity> getSimplifiedEntityList(List<Entity> entityList) {
     List<SongEntity> simplifiedEntityList = new ArrayList<>();
 
     for (Entity entity : entityList) {
       // Use round() here to set the double to 2 decimals.
       SongEntity simplifiedEntity = new SongEntity(entity.getName(), 
-          Math.round(entity.getSalience() * 100.0) / 100.0, entity.getType().toString(),
+          Double.parseDouble(salienceFormat.format(entity.getSalience())), 
+          new HashSet<String>(Arrays.asList(entity.getType().toString())),
```

Comment:
To make this a bit cleaner, you could use the method Sets.newHashSet(...) from Guave